pages/[username]/index.public.js

```diff
@@ -130,10 +147,17 @@ export default function Home({ contentListFound, pagination, userFound: userFoun
         )}
 
         <Box sx={{ width: '100%', display: 'flex' }}>
-          <Pagehead as="h1" sx={{ width: '100%', mt: 0, pt: 0, pb: 3, mb: 3 }}>
-            {userFound.username} <UserFeatures />
+          <Pagehead sx={{ width: '100%', display: 'flex', alignItems: 'center', mt: 0, pt: 0, pb: 3, mb: 3 }}>
```

Comment:
```suggestion
          <Pagehead sx={{ width: '100%', display: 'flex', alignItems: 'baseline', mt: 0, pt: 0, pb: 3, mb: 3, wordBreak: 'break-word', flexWrap: 'wrap' }}>
```

@Rafatcb, pela diferença no tamanho das fontes, acho que ficaria visualmente melhor usar `alignItems: 'baseline'`, mas daí precisa mexer também nas coins, então precisa analisar melhor.

@gabrielsozinho, obrigado pela ajuda! 💪

> Uma única observação que eu tenho é que está com um problema de responsividade em dispositivos mobile

Você fala por não quebrar a linha quando o nome de usuário somado aos novos dados não couber na largura do dispositivo, certo? Ou tem outros problemas?

Se for isso, é um problema já presente, apenas agravado com as novas informações adicionadas, mas acho que pode ser corrigido com `wordBreak: 'break-word'` e `flexWrap: 'wrap'`.

O que acham?